pkg/resources/management.cattle.io/v3/globalrole/validator.go

```diff
@@ -264,3 +269,35 @@ func (a *admitter) validateUpdateFields(oldRole, newRole *v3.GlobalRole, fldPath
 	}
 	return nil
 }
+
+func (a *admitter) validateNamespacedRules(request *admission.Request, rules map[string][]v1.PolicyRule) []error {
+	var returnErrs []error
+	// check for escalation in each namespace of the GlobalRole's NamespacedRules
+	for namespace, rules := range rules {
```

Comment:
Nit: Given that rules is already the name for a variable in this function, I would recommend changing this to something like `namespaceRules` - it makes it easier to follow what is being used where.